odoo/fields.py

```diff
@@ -356,10 +356,13 @@ def _get_attrs(self, model, name):
         # initialize ``self`` with ``attrs``
         if attrs.get('compute'):
             # by default, computed fields are not stored, computed in superuser
-            # mode if stored, not copied and readonly
+            # mode if stored, not copied and readonly.
+            # an editable computed field (store=True, readonly=False) takes its
+            # copy attribute from its class as it can be updated by users
             attrs['store'] = store = attrs.get('store', False)
             attrs['compute_sudo'] = attrs.get('compute_sudo', store)
-            attrs['copy'] = attrs.get('copy', False)
+            if 'copy' not in attrs and (not attrs['store'] or attrs.get('readonly', True)):
```

Comment:
Concerning inverse: it was on purpose as I prefer keeping them out of this fix by giving True as default for readonly instead of "not inverse" . I can make the exclusion explicit.

Reply:
The proposed fix is inconsistent with the inference of the value of `readonly` that comes after.  I propose the following rules when it comes to computed fields:
```python
# store=False unless explicitly given
attrs['store'] = attrs.get('store', False)

# readonly=not inverse unless explicitly given
attrs['readonly'] = attrs.get('readonly', not attrs.get('inverse'))

# copy=False unless explicitly given or stored and editable
if not attrs['store'] or attrs['readonly']:
    attrs['copy'] = attrs.get('copy', False)
```

Reply:
Note that it is important to force `copy=False` in some cases, instead of forcing it to `True`.  That way, one2many fields still are `copy=False` by default, even if computed editable.

Reply:
Hello,

It was the case for o2m as pointed out by removed test that "did not make any sense" according to previous comments. I did not force copy=True, as explained in commit message, and explicitly kept inverse out of the change. Anyway ok to include it if you think it is better.